src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue889.cs

```diff
@@ -12,18 +12,16 @@ public Issue889(TestDevice testDevice) : base(testDevice)
 
 	public override string Issue => "Assigning to FlyoutPage.Detail after construction doesn't work";
 
-	//[Test]
-	//[Category(UITestCategories.FlyoutPage)]
-	//[Description("Reproduce app crash - Issue #983")]
-	//public void Issue899TestsAppCrashWhenSwitchingTabs()
-	//{
-	//	App.Tap(q => q.Marked("Push new page"));
-	//	App.WaitForElement(q => q.Marked("I have been pushed"));
-	//	App.Screenshot("Push page");
-	//	App.Back();
-	//	App.Screenshot("Navigate back");
+	[Test]
+	[Category(UITestCategories.FlyoutPage)]
+	public void Issue899TestsAppCrashWhenSwitchingTabs()
+	{
+		App.WaitForElement("PushPage");
+		App.Tap("PushPage");
+		App.WaitForElement("PushedPageLabel");
+
+		App.TapBackArrow();
```

Comment:
```
D:\a\_work\1\s\src\Controls\tests\TestCases.Shared.Tests\Tests\Issues\XFIssue\Issue889.cs(23,7): error CS1061: 'IApp' does not contain a definition for 'TapBackArrow' and no accessible extension method 'TapBackArrow' accepting a first argument of type 'IApp' could be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Controls\tests\TestCases.Mac.Tests\Controls.TestCases.Mac.Tests.csproj]
D:\a\_work\1\s\src\Controls\tests\TestCases.Shared.Tests\Tests\Issues\XFIssue\Issue889.cs(23,7): error CS1061: 'IApp' does not contain a definition for 'TapBackArrow' and no accessible extension method 'TapBackArrow' accepting a first argument of type 'IApp' could be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Controls\tests\TestCases.WinUI.Tests\Controls.TestCases.WinUI.Tests.csproj]
D:\a\_work\1\s\src\Controls\tests\TestCases.Shared.Tests\Tests\Issues\XFIssue\Issue889.cs(23,7): error CS1061: 'IApp' does not contain a definition for 'TapBackArrow' and no accessible extension method 'TapBackArrow' accepting a first argument of type 'IApp' could be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Controls\tests\TestCases.Android.Tests\Controls.TestCases.Android.Tests.csproj]
D:\a\_work\1\s\src\Controls\tests\TestCases.Shared.Tests\Tests\Issues\XFIssue\Issue889.cs(23,7): error CS1061: 'IApp' does not contain a definition for 'TapBackArrow' and no accessible extension method 'TapBackArrow' accepting a first argument of type 'IApp' could be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Controls\tests\TestCases.iOS.Tests\Controls.TestCases.iOS.Tests.csproj]
    9384 Warning(s)
    4 Error(s)
```

TapBackArrow was included here https://github.com/dotnet/maui/pull/25734 but not merged yet. Include the extension method also in this PR.

Reply:
Yes, now I included the extension method in this PR.